mypy/types.py

```diff
@@ -1763,13 +1763,21 @@ class PartialType(ProperType):
     # None for the 'None' partial type; otherwise a generic class
     type = None  # type: Optional[mypy.nodes.TypeInfo]
     var = None  # type: mypy.nodes.Var
+    # For partial defaultdict[K, V], the type V (K is unknown). If V is generic,
+    # the type argument is not final and will be replaced later (it's TypeVarType
+    # or UninhabitedType).
+    #
+    # TODO: The type argument can only be TypeVarType due to leaking type variables
```

Comment:
Should we just store an erased instance here? Then we can remove this TODO and only keep the one in `checker.py`.

Reply:
A good point -- I'll try this out.